gemfirexd/tools/src/dunit/java/com/pivotal/gemfirexd/query/JSONBasicTestsDUnit.java

```diff
@@ -542,7 +542,8 @@ public void testAlterTable() throws Exception {
     assertTrue(i == 6);
   }
 
-  public void testUnsupportedClauses() throws Exception {
+  // This test should be removed as these restrictions have been removed
+  public void _testUnsupportedClauses() throws Exception {
     startVMs(1, 3, 0, null, null);
```

Comment:
JSON has been explicitly restricted so commented out the test initially. Now activated it and modified the test as per expected exception.